pkg/chrootarchive/archive.go

```diff
@@ -7,16 +7,21 @@ import (
 	"os"
 	"os/user"
 	"path/filepath"
+	"runtime"
 
 	"github.com/docker/docker/pkg/archive"
 	"github.com/docker/docker/pkg/idtools"
 )
 
 func init() {
```

Comment:
Perhaps this should be moved to `archive_unix.go` ? (I see we already have that one, so could be moved there)